packages/components/field-errors/README.md

```diff
@@ -1,55 +1,83 @@
-| -------------------- | -------- | :------: | ------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `errors`             | `object` |    -     | -      | -       | List of errors. Only entries with truthy values will count as active errors.                                                                                                                                                              |
-| `isVisible`          | `bool`   |    -     | -      | -       | `true` when the error messages should be rendered. Usually you'd pass in a `touched` state of fields.                                                                                                                                     |
-| `renderError`        | `func`   |    -     | -      | -       | Function which gets called with each error key (from the `errors` prop) and may render an error message or return `null` to hand the error handling off to `renderDefaultError`.                                                          |
-| `renderDefaultError` | `func`   |    -     | -      | -       | Function which gets called with each error key (from the `errors` prop) for which `renderError` returned `null`. It may render an error message or return `null` to hand the error handling off to `FieldError`s built-in error handling. |
+| Props                | Type     | Required | Default | Description                                                                                                                                                                                                                                                                             |
+| -------------------- | -------- | :------: | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `errors`             | `object` |          |         | List of errors. Only entries with truthy values will count as active errors.                                                                                                                                                                                                            |
+| `isVisible`          | `bool`   |          |         | `true` when the error messages should be rendered. Usually you'd pass in a `touched` state of fields.                                                                                                                                                                                   |
```

Comment:
I think this is missing default value. It might be missing in the component as well. Could we add that?